lib/avo/fields/location_field.rb

```diff
@@ -3,13 +3,14 @@
 module Avo
   module Fields
     class LocationField < BaseField
-      attr_reader :stored_as
+      attr_reader :stored_as, :zoom
 
       def initialize(id, **args, &block)
         hide_on :index
         super(id, **args, &block)
 
         @stored_as = args[:stored_as].present? ? args[:stored_as] : nil # You can pass it an array of db columns [:latitude, :longitude]
+        @zoom = args[:zoom].present? ? args[:zoom].to_i : 15
```

Comment:
I just saw on the docs that `@zoom` accept values between [0, 22] I think that would be interesting to raise an informative error about the accepted range if for some reason the value goes out of this range. @iainbeeston and @adrianthedev what do you think?

Reply:
My first thought was "let's just take the argument from the user and if it's greater than 22, then we force it as 22, but that would generate some technical support as "why isn't Avo working and not letting me add a number greater than 22".
The second thought I had was, to raise an error when that number is greater than 22. But what if the API changes at some point? We'd have to maintain that parity between our validation and the external API.
Now, I'm thinking of just leaving it without validation and directing the user towards the original library ([mapkick](https://chartkick.com/mapkick)) because of those two reasons.

So my vote goes to leaving it like that and instructing the user that we only forward the options to the original library.